core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +156,19 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.retrieveExplorationInteractionIds = function() {
+          $http.get(INTERACTIONS_BY_ID_HANDLER_URL, {
+            params: {
+              exploration_id: ctrl.explorationId
+            }
+          }).then(function(response) {
+            ctrl.explorationInteractionIds = response.data.interactions;
```

Comment:
Initialize this variable on top*

Reply:
didn't get this one

Reply:
I mean to add `ctrl.explorationInteractionIds = []` inside init method*

Reply:
done